bitcoin/src/util/psbt/mod.rs

```diff
@@ -437,6 +437,26 @@ impl PartiallySignedTransaction {
         // because there has been a new softfork that we do not yet support.
         Err(SignError::UnknownOutputType)
     }
+
+    /// Calculates fee
+    ///
+    /// ## Errors
+    ///
+    /// The function returns [`Error::MissingUtxo`] when UTXO information for at least one input
+    /// is not present or is invalid.
+    /// If calculated value is negative [`Error::NegativeFee`] is returned.
+    /// If an integer overflow happens [`Error::FeeOverflow`] is returned.
```

Comment:
This will be rendered in HTML as a single sentence, perhaps either put it as a single sentence in code
```
    /// The function returns [`Error::MissingUtxo`] when UTXO information for at least one input is
    /// not present or is invalid. If calculated value is negative [`Error::NegativeFee`] is
    /// returned. If an integer overflow happens [`Error::FeeOverflow`] is returned.
```
or as a list if that's preferred
```
    /// - [`Error::MissingUtxo`] when UTXO information for any input is not present or is invalid.
    /// - [`Error::NegativeFee`] if calculated value is negative.
    /// - [`Error::FeeOverflow`] if an integer overflow occurs.
```

In case its useful for you, here is a shell command to build the docs using our custom conditional configuration option `docsrs`:
`RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --all-features`

Reply:
appreciate your detailed suggestion, updated